packages/core/content-manager/server/services/utils/configuration/layouts.js

```diff
@@ -5,28 +5,38 @@ const { isListable, hasEditableAttribute, hasRelationAttribute } = require('./at
 
 const DEFAULT_LIST_LENGTH = 4;
 const MAX_ROW_SIZE = 12;
+const FIELD_TYPES_FULL_SIZE = ['dynamiczone', 'component', 'json', 'richtext'];
+const FIELD_TYPES_SMALL = [
+  'checkbox',
+  'boolean',
+  'date',
+  'time',
+  'biginteger',
+  'decimal',
+  'float',
+  'integer',
+  'number',
+];
+
+const isAllowedFieldSize = (type, size) => {
+  if (FIELD_TYPES_FULL_SIZE.includes(type)) {
+    return size === MAX_ROW_SIZE;
+  }
+
+  // validate, whether the field has 4, 6, 8 or 12 columns?
```

Comment:
@petersg83 I'm not sure about this, but I'm leaning towards not validating. The front-end doesn't allow values other than these (yet) and I don't see why we should validate if a user has set the column width to an arbitrary number in the database (ex: 5 / 12 and another field 7 / 12).

Wdyt?

Reply:
good to me